mathesar/tests/templatetags/test_client_files.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import importlib
+import mathesar.templatetags.client_files as client_files
+from django.template import Template, Context
+
+
+def test_client_links_development(settings):
+    settings.MODE = 'DEVELOPMENT'
```

Comment:
The standard way to use different settings per-test is to use the `@override_settings` decorator.

https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.override_settings